CoreScriptsRoot/Modules/ChatSelector.lua

```diff
@@ -0,0 +1,203 @@
+local USE_NEW_CHAT = false --and settings():GetFFlag("UseNewChat")
```

Comment:
If you use the format above you should be able to have the code in without the flag even existing, so no need to worry about updating it later.